src/auth_client_demo_assets/vue/components/LoggedOut.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="container">
+  <h1>Internet Identity Client</h1>
+  <h2>You are not authenticated</h2>
+  <p>To log in, click this button!</p>
+  <button type="button" id="loginButton" @click="login()">Log in</button>
+</div>
+</template>
+
+<script>
+import { useAuthStore } from "../store/auth";
+
+export default {
+  name: 'LoggedOut',
+  methods: {
+    login() {
+       const authStore = useAuthStore();
+      authStore.login();
+    }
+  },
+  props: {
+    msg: String
+  }
+}
+</script>
```

Comment:
The msg prop seems to be unused, this component could be simplified to simply import the store

```
<script setup>
import { useAuthStore } from "../store/auth";

const authStore = useAuthStore();
</script>
```